app/src/main/java/org/gdg/frisbee/android/api/SynchronousCallAdapterFactory.java

```diff
@@ -0,0 +1,47 @@
+package org.gdg.frisbee.android.api;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+import retrofit2.Call;
+import retrofit2.CallAdapter;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import timber.log.Timber;
+
+public class SynchronousCallAdapterFactory extends CallAdapter.Factory {
```

Comment:
Do we really need this? I mean it is good but is it needed? 
WIth Call class, we can do sync calls really easily.